packages/logo/src/Logo.stories.tsx

```diff
@@ -153,12 +153,11 @@ export const CommunityEditionLockup = (args: BaseLogoProps) =>
   Template(CommunityEditionLogoLockup, args);
 export const UniversityLockup = (args: BaseLogoProps) =>
   Template(UniversityLogoLockup, args);
+export const AtlasMark = (args: BaseLogoProps) => Template(AtlasLogoMark, args);
+AtlasMark.storyName = 'Atlas Mark';
```

Comment:
Thinking about this more, this shouldn't be the `AtlasMark`. This is an updated `Atlas` logo

Reply:
Also, I believe all the `...Lockup` logos are deprecated (@sandynguyenn can you confirm? )